index.html

```diff
@@ -6,5 +6,6 @@
 <body>
 <h1>Hello Word !</h1>
 <p>voila voila</p>
```

Comment:
oui bon, rien de passionnant quoi...
```suggestion
<p>voila voila quoi</p>
```